packages/neuron-wallet/src/models/system-script-info.ts

```diff
@@ -14,9 +15,9 @@ export default class SystemScriptInfo {
   static MULTI_SIGN_HASH_TYPE = ScriptHashType.Type
 
   static DAO_SCRIPT_HASH = new Script(
-    SystemScriptInfo.DAO_CODE_HASH,
+    systemScripts.NERVOS_DAO.codeHash,
```

Comment:
To remove `@nervosnetwork/ckb-sdk-utils` finally, I propose we'd better use the process.env to create the DAO script.

Reply:
will replace `systemScripts` with lumos in another PR